src/main/java/step2/service/LottoStatisticsService.java

```diff
@@ -0,0 +1,34 @@
+package step2.service;
+
+import java.util.Map;
+
+import step2.domain.LottoOrder;
+import step2.domain.Rank;
+
+/**
+ * 로또 통계 서비스
+ */
+public class LottoStatisticsService {
+    private final LottoOrder order;
+    private final Map<Rank, Integer> result;
+
+    public LottoStatisticsService(LottoOrder order, Map<Rank, Integer> result) {
+        this.order = order;
+        this.result = result;
+    }
+
+    public double getRate() {
+        int spend = order.getLottoList().getTotal();
+        int earn = getTotalCount();
+        return Math.round(earn * 100 / spend) / 100.0 ;
+    }
+
+    public int getTotalCount() {
+        return result.entrySet()
+            .stream()
+            .mapToInt((entry) -> {
+                return entry.getKey().getPrize() * entry.getValue();
+            })
```

Comment:
```suggestion
            .mapToInt((entry) -> {
                return entry.getKey().계산?(entry.getValue());
            })
```
값을 꺼내기보단 메시지를 보내보는건 어떨까요?

Reply:
요구사항들을 보며 메시지를 보낸다는 말을 잘 이해를 못했는데, 이해가 되었습니다. 감사합니다!